rules/sonic-yang-mgmt-py2.mk

```diff
@@ -3,6 +3,6 @@
 SONIC_YANG_MGMT_PY2 = sonic_yang_mgmt-1.0-py2-none-any.whl
 $(SONIC_YANG_MGMT_PY2)_SRC_PATH = $(SRC_PATH)/sonic-yang-mgmt
 $(SONIC_YANG_MGMT_PY2)_PYTHON_VERSION = 2
-$(SONIC_YANG_MGMT_PY2)_DEBS_DEPENDS = $(LIBYANG)
+$(SONIC_YANG_MGMT_PY2)_DEPENDS = $(SONIC_YANG_MODELS_PY3)
```

Comment:
Are you able to build if libyang package is not built with this change?

Reply:
Yes, because SONIC_YANG_MGMT_PY2 depends on LIBYANG, so it will build libyang if not built already. But yeah if your concern is that,  libyang pkg does not present somehow, then i will not pass and I need to keep both libyang and SONIC_YANG_MGMT_PY2 dependencies.

Reply:
Done